docs/courses/interactive_modeling.rst

```diff
@@ -0,0 +1,102 @@
+.. _interactive_modeling:
+
+Interactive Modeling
+====================
+
+Physical System
+---------------
+
+Giant axon from the squid: Loligo pealei
+
+Conceptual Model
+----------------
+
+Hodgkin-Huxley cable equations
+++++++++++++++++++++++++++++++
+
+.. image:: img/hh.gif
+    :align: center
+
+Simulation
+----------
+
+Computational implementation of the conceptual model
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+We *could* implement this model in Python:
+
+.. code::
+    python
+
+    from neuron import h, gui
+
+    axon = h.Section(name='axon')
+    axon.L = 2e4
+    axon.diam = 100
+    axon.nseg = 43
+    axon.insert('hh')
+
+**But** for this exercise, let's instead use the CellBuilder tool to create the model:
+
+.. image:: img/cbgeom.gif
+    :align: center
+
+.. image:: img/cbbio.gif
+    :align: center
+
+Save the model in ``hhaxon.ses`` using :menuselection:`NEURONMainMenu --> File --> savesession`
+
+
+Using the computational model
++++++++++++++++++++++++++++++
+
+If starting from a fresh launch of python, you can load the saved ses file by loading NEURON and its GUI: ``from neuron import h. gui`` and then selecting :menuselection:`NEURONMainMenu --> File --> loadsession`
+
+Alternatively you can use NEURON to execute ``hhaxon.ses``
+
+1. Change to the appropriate directory in your terminal
+
+2. Start python, and at the >>> prompt enter the commands
+
+.. code::
+    python
+
+    from neuron import h, gui
+    h.load_file('hhaxon.ses')
+
+
+Exercises
+---------
+
+1. Stimulate with current pulse and see a propagated action potential.
```

Comment:
If you start the content on a new, indented line, it'll correctly align everything that should be under the 1.